feat(Transition): Add `direction` to `slide` Transition (#5466)
sveltejs/svelte: site/content/docs/03-run-time.md

* `delay` (`number`, default 0) — milliseconds before starting
* `duration` (`number`, default 400) — milliseconds the transition lasts
* `easing` (`function`, default `cubicOut`) — an [easing function](docs#svelte_easing)

> * `direction` (`string`, default `vertical`) — direction of transition, vertical or horizontal

Reviewer: ```suggestion
* `direction` (`string`, default `vertical`) — direction of transition, `"vertical"` or `"horizontal"`
```

> * `direction` (`string`, default `vertical`) — direction of transition, `"vertical"` or `"horizontal"`

Reviewer 2: ```suggestion
* `direction` (`string`, default `vertical`) — direction of transition, `'vertical'` or `'horizontal'`
```

</script>

{#if condition}

> 	<div transition:slide="{{delay: 250, duration: 300, easing: quintOut, direction: 'vertical' }}">

Reviewer: probably use `horizontal` as it is not the default value

```suggestion
	<div transition:slide="{{delay: 250, duration: 300, easing: quintOut, direction: 'horizontal' }}">
```
